day7/TreeMapEx.java

```diff
@@ -0,0 +1,26 @@
+package day7;
+
+import java.util.TreeMap;
+
+public class TreeMapEx {
```

Comment:
Your TreeMapEx class is mostly correct and demonstrates several operations on a TreeMap, such as adding, copying, removing, and replacing elements. However, there are a couple of improvements and fixes that you can make:


**Incorrect Type in getOrDefault:**

The getOrDefault method's default value type must match the value type of the TreeMap. Since your TreeMap is of type <String, Integer>, the default value should be an Integer, not a String.

**General Improvements:**


You might consider adding comments or explanations for what each operation does for clarity, especially when sharing or using this code as an educational example.